src/Home_Work/Lesson_2.java

```diff
@@ -0,0 +1,58 @@
+
+package Home_Work;
+
+public class Lesson_2 {
+    public static void main(String[] args) {
+        IntSum(12, 4);
+        IntegerCheck(-2);
+        IntegerReturn(33);
+        WriteInt("GeekBrains", 5);
+        Year();
+
+    }
+
+    public static boolean IntSum(int a , int b) {
+        int summa = a + b;
+        if (summa >= 10 && summa <= 20) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
поскольку логическое выражение само по себе возвращает boolean результат, то можно упростить:
```suggestion
        return (summa >= 10 && summa <= 20);
```